pxr/usdValidation/usdUtilsValidators/validators.cpp

```diff
@@ -143,6 +144,101 @@ _FileExtensionValidator(const UsdStagePtr& usdStage) {
     return errors;
 }
 
+static UsdValidationErrorVector
+_GetUsdzPackageErrors(const SdfLayerHandle &rootLayer)
+{
+    const UsdZipFile zipFile = UsdZipFile::Open(rootLayer->GetRealPath().c_str());
+    if (!zipFile)
```

Comment:
@tallytalwar I noticed testing locally that when passing a .usda file in here, this if statement does not filter out the .usda layer, which I anticipated that it would. All of the errors are still coming back correctly but I wanted to ask if there was some other condition to put here to exit out early as I couldn't find something on the UsdZipFile class that was applicable

Reply:
Yeah looks like from the code it works even if the layer is a usda layer, as it goes through ArAsset. I will confirm internally if thats anticipated.